rich/_log_render.py

```diff
@@ -8,29 +8,31 @@
     from .console import Console, ConsoleRenderable, RenderableType
     from .table import Table
 
+TimeFormatterCallable = Callable[[datetime], Text]
```

Comment:
Could we call it `FormatTimeCallable`? I like to start callables with a verb.

Reply:
Sounds good, I'll rename and push